.github/ISSUE_TEMPLATE/deprecation.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Deprecation
+about: For the Octopus team to make customers aware of upcoming deprecations.
+title: ''
+labels: tag/breaking-change
```

Comment:
I don't think we want to default to breaking change. As discussed this morning, adding the deprecation shouldn't effect behaviour at all.

I can't really see any labels that we *do* want though; maybe we should have a `kind/deprecation` label?

Reply:
Could also just not have any labels at all. This is primarily for release notes after all.

Reply:
Yeh, I was just generally asking the question. I don't know if having a label is important for anyone. I don't think it is for us, so I'd be happy not to have any in the template.